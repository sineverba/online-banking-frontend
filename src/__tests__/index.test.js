import index from "../index";

describe("Test index.js", () => {
  it("Should render app without crashing", () => {
    expect(
      JSON.stringify(
        Object.assign({}, index, { _reactInternalInstance: "censored" })
      )
    ).toMatchSnapshot();
  });
});
