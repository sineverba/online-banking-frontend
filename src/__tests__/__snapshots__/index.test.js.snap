// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test index.js Should render app without crashing 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;
